tests/honey_badger.rs

```diff
@@ -1,11 +1,10 @@
 #![deny(unused_must_use)]
 //! Network tests for Honey Badger.
 
-mod network;
+pub mod net;
```

Comment:
Is the `pub` to avoid the "unused" warnings?

Reply:
Yes, that seems to be the strategy the already ported tests are following, like the binary_agreement test.